frontend/src/Frontend.hs

```diff
@@ -43,9 +43,9 @@ frontend = Frontend
       el "title" $ text "Taut - Slack Archive Viewer"
       -- FIXME: This throws JSException
       -- el "title" $ subRoute_ $ \case
-      --   Route_Home -> text "Taut"
-      --   Route_Messages -> do
-      --     r :: Dynamic t Day <- askRoute
+      --   FrontendRoute_Home -> text "Taut"
+      --   FrontendRoute_Search -> do
+      --     r :: Dynamic t (PaginatedRoute UTCTime Text) <- askRoute
       --     text "Taut - "
       --     dynText $ fmap (T.pack . show) r
```

Comment:
@ali-abrar Uncommenting this code still throws JSException. Perhaps the hydration fix is not complete?

Reply:
Also I noticed a minor regression in this upgrade. Accessing the app through http://localhost:8000/ no longer works; seeing these jsaddle errors on the console:

![image](https://user-images.githubusercontent.com/3998/57929737-21d1f400-7882-11e9-8f8e-d0b21a17aaff.png)

Using http://127.0.0.1:8000/ works. I thought `ob run` has been respecting config/common/route - but apparently not anymore.